nixio/dimensions.py

```diff
@@ -466,8 +478,44 @@ def create_new(cls, data_array, index, ticks):
             newdim._h5group.write_data("ticks", ticks, dtype=DataType.Double)
         return newdim
 
+    @property
+    def is_alias(self):
+        """
+        In versions < 1.5 a RangeDimension that was referring to the DataArray it describes
+        was called an AliasRangeDimension. This has been made more flexible by allowing to link
+        to any DataArray or DataFrame.
+        For downward compatibility this method has been re-introduced in 1.5.1.
+
+        :returns: True, if this dimension links to a DataArray, False otherwise
+        :rtype: bool
+        """
+        if self._h5group.has_data("ticks"):
+            return False
+        elif not self.has_link and len(self._h5group) > 0:
+            return True
+        elif self.has_link and self.dimension_link._data_object_type == "DataArray":
+            return True
+        return False
```

Comment:
Does alias dimension imply it's a self-referencing dimension?  Should this be checking if the linked DataArray is the same as the DataArray that owns the dimension?

Reply:
Yes, thought about checking for it but then thought that it actually does not matter for the idea ...
What do you think? Better to keep it strictly in the "old" sense of the aliasRangeDimension?

Reply:
I guess it depends what we want "alias" to mean.  It used to be self reference only but we could say now it means "link to a DA".  I have no strong feelings either way.

Reply:
Since this will die with the use of the AliasRangeDimension, I would say there's no harm done and would argue for keeping it simple..., i.e. as it is now :)